test/e2e/specs/specs-playwright/wp-likes-spec.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import {
+	BrowserHelper,
+	DataHelper,
+	LoginFlow,
+	NewPostFlow,
+	GutenbergEditorPage,
+	LikesComponent,
+} from '@automattic/calypso-e2e';
+
+/**
+ * Constants
+ */
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const host = DataHelper.getJetpackHost();
+const viewportName = BrowserHelper.getViewportName();
+const quote =
+	'The foolish man seeks happiness in the distance. The wise grows it under his feet.\n— James Oppenheim';
+
+describe( `[${ host }] Likes: (${ viewportName }) @parallel`, function () {
+	this.timeout( mochaTimeOut );
+
+	describe( 'New post', function () {
+		let gEditor;
+		let likesComponent;
+
+		before( 'Can log in', async function () {
```

Comment:
I think it would be a good moment to decide on a test naming convention. Personally, I would drop the use of `Can` / `Should` / etc. and just go with the step-like description: 

`Log in` → `Start new post` → `Enter post title` → etc.

Also, I vaguely remember there were some issues with errors thrown from the before/after hooks, so it might be better to keep it as an `it` call. I might be mixing Jest with Mocha here though so calling @scinos for clarification! 🙏

Reply:
Agreed on the first point about naming convention for tests. Let's drop the `modal verbs`.

On the second point, in my experience so far there appears to be no issue with `mocha` when the before hook is placed in the test file. This placement ensures that `before` hook runs only once immediately preceding the inner `describe` block.

Whether to use the `before` hook was something I toyed with for some time before settling on the current approach, which is:
- if the login process is what is being tested (eg. wp-log-in-out spec) then use `it`.
- if the login process is not what is being tested (eg. likes) then use a hook to ensure it completes.

But as I typed out the previous bullets it occurred to me there is only a minor distinction between the two, really. So changing this to an `it` step could eliminate one consideration for test writers.

Reply:
Replaced `before` with `it` to treat the login step as just another test step, and removed modal verbs from step names.

Reply:
Thanks for elaborating! The tested vs not-tested distinction does make a lot of sense to me, haven't thought about it this way before. Sounds good to me whichever way it goes 👍